doc/sphinx/source/tutorials/run-qed-fit.rst

```diff
@@ -3,32 +3,232 @@
+  [INFO]: Downloading https://data.nnpdf.science/photons/photon_theoryID_702_fit_251127-jcm-lh-qed-002.tar.
+  [==================================================] (100%)
+  [INFO]: Extracting archive to /user/envs/nnpdf/share/NNPDF/photons_qed
+  PosixPath('/user/envs/nnpdf/share/NNPDF/photons_qed/photon_theoryID_702_fit_251127-jcm-lh-qed-002')
 
-Whenever the photon PDF is generated, it will remain constant during the fit and will be considered in the momentum sum rule.
+As in the case of ``vp-setupfit``, this will download and extract the photon PDF
+set in the local ``photons_qed_path`` specified in the :ref:`profile file <nnprofile>`.
+Once the photon PDF set is available locally, the user can proceed to run the
+fit with ``n3fit`` as usual.
 
 .. warning::
 
-   At the moment it is not possible to run QED fits in parallel, as the FiatLux library cannot run in parallel.
+   If ``vp-setupfit`` is not run before ``n3fit``, and the photon PDF set is not
+   available locally, the code will **not** attempt to download it from the server,
+   but will directly proceed to compute it on the fly. See the :ref:`next section <generating-photon-sets>` for
+   more details.
+
+.. _generating-photon-sets:
+Generating new Photon PDF sets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If the desired photon PDF set is not available locally nor on the NNPDF server, the
+code will generate the required photon PDF set replicas on the fly using `FiatLux <https://github.com/scarrazza/fiatlux/>`_.
+This can be done either during the ``vp-setupfit`` step, which precomputes all photon
+replicas before starting the fit, or during the fitting step with ``n3fit``, which
+generates the photon replica as needed for each replica being fitted.
+
+.. note::
+
+  Generating photon PDF sets on the fly can add significant overhead to the
+  fitting procedure. It is therefore strongly advised to precompute the photon
+  PDF set using ``vp-setupfit`` before running the fit with ``n3fit``.
+
+In either case, the generated photon PDF set will be stored locally in the
+``photons_qed_path`` specified in the :ref:`profile file <nnprofile>`, so that
+it can be reused in future fits. The folder containing the photon replicas will
+be named as ``photon_theoryID_<theoryid>_fit_<luxset>`` where ``<theoryid>`` and
+``<luxset>`` are the values specified in the runcard. The folder contains a ``npz``
+file for each photon replica, named as ``replica_<replica_id>.npz``. Each replica
+file contains the photon PDF grid computed with FiatLux at :math:`Q_{\rm init} = 100` GeV,
+prior to the evolution through EKO.
+
+.. important::
+
+  Automatic upload to the NNPDF server through ``vp-upload`` is **not**
+  supported at the moment. The user should manually create a ``tar`` archive
+  file containing the photon replicas and upload it to server. Refer to the
+  :ref:`profile file <nnprofile>` to find the remote URL where photon PDF sets are stored.
```

Comment:
A few comments here
1) you should mention explicitly the command to run for the `.tar` because it is not the same to do

```
tar -cf my_photon.tar my_photon_folder
```

or
```
cd my_photon_folder
tar -cf my_photon.tar *.npz
```

or even
```
cd my_photon_folder
tar -czf my_photon.tgz *.npz
```

2) In the profile you have the URL but you didn't add a `photon_target_dir` so it is impossible to know where in the server the photon lives!